Sources/StreamChat/Models/Channel.swift

```diff
@@ -63,7 +63,16 @@ public struct _ChatChannel<ExtraData: ExtraDataTypes> {
     /// - Important: This list doesn't have to contain all members of the channel. To access the full list of members, create
     /// a `ChatChannelController` for this channel and use it to query all channel members.
     ///
-    public let cachedMembers: Set<_ChatChannelMember<ExtraData.User>>
+    @available(*, renamed: "lastActiveMembers")
+    public var cachedMembers: Set<_ChatChannelMember<ExtraData.User>> { lastActiveMembers }
```

Comment:
Can you please move this to some kind of extension that we can then easily delete once we decide to drop this?